src/v0/destinations/kafka/transform.js

```diff
@@ -76,21 +76,6 @@ const process = event => {
     filterConfigTopics(message, destination) ||
     destination.Config?.topic;
 
-  // TODO: uncomment this when v.1.3.0 of server is avialble in all envs
```

Comment:
Revert this and refactor it. CC @utsabc

Reply:
@utsabc do i need to refactor and uncomment it or keep it commented??

Reply:
done